Release-Notes/Release2.md

```diff
@@ -0,0 +1,25 @@
+# Release Notes for SCS Release 2
+(Release Date: 2022-03-XX)
+
+## Scope
+
+Main goals for Release 2 (R2) was ...
+
+## Component Versions
+
+* Kubernetes 1.22.2
+* Kubernetes Cluster API 1.0.0
```

Comment:
1.0.2

Reply:
Should we merge this now and then have PRs against the main branch? Or PRs against a branch which gets merged to main when the release approaches?

Reply:
As someone might think that R2 already has been released, I would suggest working either directly on the branch or with PR against this branch.